org.github.xournalpp.xournalpp.yaml

```diff
@@ -0,0 +1,88 @@
+---
+app-id: com.github.xournalpp.xournalpp
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: xournalpp
+rename-icon: xournalpp
+rename-desktop-file: xournalpp.desktop
+rename-appdata-file: xournalpp.xml
+finish-args:
+  # X11 + XShm access
+  - "--share=ipc"
+  - "--socket=x11"
+  # Wayland access
+  - "--socket=wayland"
+  # Needs to save files locally
+  - "--filesystem=xdg-documents"
+  - "--socket=pulseaudio"
+  # Allow dconf
+  - "--filesystem=xdg-run/dconf"
+  - "--filesystem=~/.config/dconf:ro"
+  - "--talk-name=ca.desrt.dconf"
+  - "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+build-options:
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/man"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "/share/man"
+  - "/share/pkgconfig"
+  - "*.la"
+  - "*.a"
+modules:
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+      - "-DCMAKE_INSTALL_INCLUDEDIR=/app/include"
+      - "-DENABLE_LIBOPENJPEG=none"
+    cleanup:
+      - "/bin"
+    sources:
+      - type: archive
+        url: https://poppler.freedesktop.org/poppler-0.69.0.tar.xz
+        sha256: 637ff943f805f304ff1da77ba2e7f1cbd675f474941fd8ae1e0fc01a5b45a3f9
+  - name: libportaudiocpp
+    buildsystem: autotools
+    config-opts:
+      - "--enable-cxx" # compile with c++ headers!!!
+    sources:
+      - type: git
+        url: https://git.assembla.com/portaudio.git
+        commit: 396fe4b6699ae929d3a685b3ef8a7e97396139a4
+    cleanup:
+      - "/include"
+      - "/lib/*.a"
+      - "/lib/*.la"
+      - "/lib/pkgconfig"
+      - "/man"
+      - "/share/aclocal"
+      - "/share/doc"
+      - "/share/gtk-doc"
+      - "/share/man"
+      - "/share/pkgconfig"
+  - name: libsndfile
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://www.mega-nerd.com/libsndfile/files/libsndfile-1.0.27.tar.gz
+        sha256: a391952f27f4a92ceb2b4c06493ac107896ed6c76be9a613a4731f076d30fac0
+  # Tried git, but failed to get the autogen script to run.
+  #     - type: git
+  #       url: https://github.com/erikd/libsndfile
+  #       commit: 1d928bffdb65e827c3e7d17ece0e123b4a70ec6c
+  #       tag: 1.0.28
```

Comment:
A good way to find out how to build a library is looking at the PKGBUILD of Arch packages.
You might want to look at the dependencies as there could be a change to 1.0.27.

Reply:
Thx, nice tip. But the do exactly the same as i do. Download the .tar.gz from www.mega-nerd.com/libsndfile instead of using the git :)

Reply:
That was not to point out that you have to use git ;-) It was more a general tip, as you pointed out that you did not get a working build setup.

But for your information there is a package that uses git:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=libsndfile-git